korman/operators/op_toolbox.py

```diff
@@ -170,7 +170,40 @@ def execute(self, context):
         for i in bpy.data.objects:
             i.plasma_object.enabled = self.enable
         return {"FINISHED"}
+        
+class PlasmaToggleDoubleSidedOperator(ToolboxOperator, bpy.types.Operator):
+    bl_idname = "mesh.plasma_toggle_double_sided"
+    bl_label = "Toggle All Double Sided"
+    bl_description = "Toggles all meshes to be double sided"
+    
+    enable = BoolProperty(name="Enable", description="Enable Double Sided")
+    
+    def execute(self, context):
+        enable = self.enable
+        for mesh in bpy.data.meshes:
+            if mesh is None:
+                continue
+            for material in mesh.materials:
+                if material is None:
+                    continue
```

Comment:
I don't think this will actually filter out regions properly. It might be best to just remove it so the operator "does what it says"(TM).